x-pack/plugins/ml/public/application/components/annotations/annotation_flyout/_index.scss

```diff
@@ -0,0 +1,5 @@
+.mlAnnotationFlyout {
+  &:focus {
+    outline: none;
+  }
+}
```

Comment:
As @ryankeairns  said, this will be fixed via EUI before 7.14. So I'd recommend not doing any overrides.

Reply:
Thanks @ryankeairns and @cchaos for the context 🙏 Good to know the issue will be fixed in 7.13. I have removed the css overrides here [`baea0c3` (#98233)](https://github.com/elastic/kibana/pull/98233/commits/baea0c3354d2a6c0811b120f4c2f942c5b0b395f)